app/assets/stylesheets/custom.css.scss

```diff
@@ -141,28 +135,28 @@ h1 {
   background-color: darkgrey;
 }
 
-// table td {
-//   font-family: monospace;
-//   border: 2px solid black;
-//   padding: 10px;
-//   align: center;
-// }
-//
-// .admin_orders_list {
-//   font-family: monospace;
-//   width: 65%;
-//   padding: 5px;
-//   border: 1px solid;
-//   height: 300px;
-//   overflow: auto;
-// }
-//
+table td {
+  font-family: monospace;
+  border: 2px solid black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.